utils/s2n_init.c

```diff
@@ -133,3 +133,8 @@ static void s2n_cleanup_atexit(void)
 {
     (void) s2n_cleanup_atexit_impl();
 }
+
+bool s2n_is_init(void)
```

Comment:
```suggestion
bool s2n_is_initialized(void)
```
Why not?

Reply:
I was trying to match `s2n_init`, but I can do the full word.